### PR TITLE
Insure IDCTAB is current in FLT files

### PR DESCRIPTION
**Pull request by stsci-hack**
The IDCTAB in the FLT/FLC files have reverted to older IDCTABs in the archive.  This implements support for a new environment variable, **PIPELINE_RESET_IDCTAB**, to enable this new function, `reset_idctab_kw`,  to sync the IDCTABs (and corresponding PRIMARY WCS) in the FLT/FLC files with the newest IDCTAB specified in the RAW file header.

### CODE REVIEW

File: drizzlepac/runastrodriz.py [hunk 1]
```diff
@@ -1233,6 +1253,48 @@ def restore_pipeline_default(files):
...
+        newmsg = "Updating IDCTAB {} in {}\n".format(flt_idctab, flt)
+        if logfile:
+            # Write message out to temp file and append it to full trailer file
+            _updateTrlFile(logfile, newmsg)
+        else:
+            print(newmsg)
+
+        # Get info from all hdrlet extensions in file
+        wnames = headerlet.get_headerlet_kw_names('j8mbnlkbq_flc.fits', 'wcsname')
+        hnames = headerlet.get_headerlet_kw_names('j8mbnlkbq_flc.fits', 'hdrname')
```
Comment (mdlpstsci) [inline drizzlepac/runastrodriz.py:1288]: Somehow I do not think you meant to hardcode this explicit filename in this method to get the wname and hnames!

Comment (stsci-hack) [reply drizzlepac/runastrodriz.py]: Like a broken clock, it would at least be right 1 time during reprocessing.  
Fixed in f6ffd00.

**Review by mdlpstsci — APPROVED** [review]
Changes look fine.

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Oops - tests found.  If line 228 is not true, then the variable "reset_idctab_switch" is not set.

**Comment by stsci-hack** [comment]
Problem with tests should be resolved with 141ca84.

**Review by mdlpstsci — APPROVED** [review]
Looks good now.  Sorry to iterate as I should have caught this lack of initialization in the first place.

**Pull request merged**